path: [ [0,0], [9.20902518713, 0.178714026039], [7.92034779231, 18.1181837393], [-14.0958404079, 15.0337042594], [-13.0047213288, -2.79084742571], [1, 0]]
#path: [ [0,0], [8.20902518713, 0.178714026039], [6.92034779231, 18.1181837393], [-15.0958404079, 15.0337042594], [-10.6481745979, 7.74023300642], [-13.0047213288, -2.79084742571], [0, 0]]
    
step_size: .1
path_frame: odom
odom_frame: odom
